gulpfile.js

```diff
@@ -61,7 +61,7 @@ function test()
 }
 
 function build()
-{
+{console.log('aa');
```

Comment:
@allmywant what is the reason for this `console.log`?

Reply:
@dapperdrop 
That was the debug code I made and now I've removed it.

Reply:
Thanks @allmywant